app/models/user.rb

```diff
@@ -459,6 +466,14 @@ def set_projects_limit
     self.projects_limit = current_application_settings.default_projects_limit
   end
 
+  def set_current_streak
```

Comment:
Do we really need to do this? What about setting a default value in the database of `0`?